### ISSUE TITLE
[docs][a11y] Add alt text to Typography page image

### ISSUE THREAD

**Issue by hellogreg** [RedHat-UX/red-hat-design-system#1689]
### Description

The image in the [Typography page's Style section](https://ux.redhat.com/foundations/typography/#style) is missing alt text.

Something like this would do: `alt="artwork demonstrating Red Hat Display and Red Hat Text at different font weights"`
 
![alt-text-less image from the Typography page](https://github.com/RedHat-UX/red-hat-design-system/assets/642695/c414ad61-aa23-4600-a529-5564abfc7714)
